doc/docs/Python_Tutorials/Near_to_Far_Field_Spectra.md

```diff
@@ -662,7 +662,7 @@ if __name__ == "__main__":
 
 Note: in the case of a disc, the set of dipoles within the quantum well (QW) which spans a 2D surface only needs to be computed along a line. This means that the number of single-dipole simulations necessary for convergence is the same in cylindrical and 3D Cartesian coordinates.
 
-Note: for randomly polarized emission from the QW, each dipole requires computing the emission from the two orthogonal "in-plane" polarization states of $E_r$ and $E_\phi$ separately and averaging the results in post processing. In this example, only the $E_r$ polarization state is used.
+Note: for randomly polarized emission from the QW, each dipole requires computing the emission from the two orthogonal "in-plane" polarization states of $E_r$ and $E_\phi$ separately and averaging the Poynting flux in post processing. (The averaging is based on the principle that the emission from dipoles with different polarizations, whether the dipoles are positioned at the same or different locations, is incoherent.) In this example, only the $E_r$ polarization state is used.
```

Comment:
This is true for an isotropic polarizability; more generally, you may need to compute the emission from the principal axes of the polarizability.

Reply:
Fixed.